perf/benchmark_dump.jl

```diff
@@ -15,21 +15,18 @@ using CUDA
 import ClimaComms
 using Plots
 using PrettyTables
+import YAML
 
-s = CA.argparse_settings()
-parsed_args = CA.parse_commandline(s);
-output_dir = joinpath(parsed_args["job_id"])
-
-# Set non-varying arguments
-parsed_args["z_elem"] = 50
-parsed_args["dt"] = "50secs"
+parsed_args = CA.parse_commandline(CA.argparse_settings())
+config_dict = merge(YAML.load_file.(parsed_args["config_file"])...)
+output_dir = joinpath(config_dict["job_id"])
```

Comment:
Where is the toml for this job?

Reply:
The toml for this job is [here](https://github.com/CliMA/ClimaAtmos.jl/blob/82f828cd29a0cc51e97b60299ef826e3fa039a5d/.buildkite/perf_configs/gpu_implicit_barowave_wrt_h_elem.yml) and it's used in the pipeline [here](https://github.com/CliMA/ClimaAtmos.jl/blob/82f828cd29a0cc51e97b60299ef826e3fa039a5d/.buildkite/pipeline.yml#L893-L899).

Reply:
Ah, ok, thanks, I just didn't grep it properly.